helper-cli/build.gradle.kts

```diff
@@ -52,6 +53,16 @@ repositories {
             includeGroup("org.gradle")
         }
     }
+
```

Comment:
Reminder: Probably not necessary anymore when rebasing onto `master` after https://github.com/oss-review-toolkit/ort/pull/3235 got merged.